zeek/README.md

```diff
@@ -0,0 +1,195 @@
+    ```
+    logs_enabled: true
+    ```
+
+2. Add this configuration block to your zeek.d/conf.yaml file to start collecting your Zeek logs.
+    ```
+    logs:
+    - type: file
+        path: /opt/zeek/logs/current/*.log
+        exclude_paths:
+          - /opt/zeek/logs/current/*.*.log
+        service: zeek
+        source: zeek
+    ```
+**Note:** Ensure to include the log file's paths within the `exclude_paths` parameter to prevent the ingestion of unsupported or undesired log files during the monitoring process.
+  
+eg.
+  ```
+  exclude_paths:
+    - /opt/zeek/logs/current/ntlm.log
+    - /opt/zeek/logs/current/radius.log
+    - /opt/zeek/logs/current/rfb.log
+  ```
+
+3. [Restart the Agent][1].
+
+**Corelight Zeek:**
+1. Collecting logs is disabled by default in the Datadog Agent. Enable it in datadog.yaml:
+    ```
+    logs_enabled: true
+    ```
+
+2. Add this configuration block to your zeek.d/conf.yaml file to start collecting your logs
+    ```
+    logs:
+    - type: tcp
+      port: <PORT>
+      service: corelight
+      source: zeek
+    ```
+
+3. [Restart the Agent][1].
+
+4. Configuring Syslog Message Forwarding from corelight
+    1. Access the Corelight Sensor UI:  
+       - Open a web browser and navigate to the IP address or hostname of your Corelight sensor.
+       - Log in with your administrative credentials.
+    2. Navigate to the Zeek Configuration Page:  
+       The exact path may vary depending on your sensor's firmware version. Look for options related to "Zeek" or "Logging".  
+       Common paths include:  
+        - Settings > Logging
+        - Configuration > Zeek > Logging
+    3. Enable Syslog Output:
+        - Locate the option to enable syslog output for Zeek logs.
+        - Select the checkbox or toggle to activate it.
+    4. Specify Syslog Server Details:
+       Provide the following information:
+       - Syslog server IP address: The destination where you want to send the Zeek logs.
+       - Syslog port: The port on which the syslog server is listening (typically 514).
+       - Facility: The syslog facility to use.
+       - Severity level: The minimum severity of events to send.
+    5. Save and Apply Changes:  
+       - Click the "Save" or "Apply" button to commit the configuration changes.
+
+
+### Validation
+
+[Run the Agent's status subcommand][2] and look for `zeek` under the Checks section.
+
+## Data Collected
+
+### Metrics
+
+The zeek integration does not include any metrics.
+
+### Events
+
+The zeek integration does not include any events.
+
+### Service Checks
+
+The zeek integration does not include any service checks.
+
+## Troubleshooting
+
+**Opensource Zeek:**
+
+If you see a **Permission denied** error while monitoring the log files, see the following instruction
+
+   1. Give read permission to dd-agent user to monitor the log files
+      ```
+      sudo chown -R dd-agent:dd-agent /opt/zeek/current/
```

Comment:
```suggestion
  sudo chown -R dd-agent:dd-agent /opt/zeek/current/
```

Reply:
We have kept it in the ```shell ``` block because this is a linux command.